spec/live/push_notification_device_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'core/spec_helper'
+
+describe ZendeskAPI::PushNotificationDevice do
+  describe ".destroy_many" do
+    describe "Existing push notification devices" do
+      it "destroys the given push notification devices" do
+        VCR.use_cassette("push_notification_devices_destroy_many") do
+          ZendeskAPI::PushNotificationDevice.destroy_many(client, ["foo", "bar"])
+        end
+      end
+    end
+
+    describe "Non-existing devices" do
+      it "silently ignores the devices" do
+        VCR.use_cassette("push_notification_devices_destroy_many") do
+          ZendeskAPI::PushNotificationDevice.destroy_many(client, ["baz"])
+        end
```

Comment:
These tests don't really test anything, since it just returns a collection. You can remove this entire test if you want, we don't test absolutely every endpoint.